cigaradvisor/styles/styles.css

```diff
@@ -434,6 +443,74 @@ color: #b19b5e;
   font-weight: 700;
 }
 
+main .section>.default-content-wrapper>h2{
+  text-align: center;
+  text-transform: uppercase;
+  font-family: var(--ff-montserrat);
+  font-weight: 900;
+  color: #8e7b5c;
+  padding: 38px 15px;
+  font-size: 45px;
+}
+
+main .section>.default-content-wrapper>h2 ::before{
+  content: 'decor';
+    color: transparent;
+    background: url('/cigaradvisor/images/homepage/sectiontitle/media_10c44610cd58523f02bbd64b114069404ca081e41.png') center no-repeat;
```

Comment:
if it's in the CSS - it should probably come from the codebase, not from the media-bus